src/routes/commands/commands_resource.ts

```diff
@@ -167,27 +166,27 @@ export const commands: Command[] = [
         tooltip: "Display information about the discord server",
         quick: "-mod server",
         description:
-            "By using this command, you can view the creation date of the server, " +
-            "the owner, member count and the emojis available.",
+            "This command provides the creation date of the server, " +
+            "the owner, member count and the available emoji.",
         usages: ["-mod server"],
         examples: ["-mod server"]
     },
     {
         name: "Invite",
-        tooltip: "Display invite links for Jim",
+        tooltip: "Display links to invite Jim",
```

Comment:
The reason I initially wrote "Display invite links for Jim" is because there are two links, one is to invite jim and one is to invite people to the support server so it's not exactly right to change it to "Display links to invite Jim", this can be kept as is.

Reply:
Will fix